components/MatchByColor/MatchByColorGame.js

```diff
@@ -14,6 +14,7 @@ import randomstring from 'random-string';
 import AnimatedSprite from '../AnimatedSprite/AnimatedSprite';
 import HomeButton from '../HomeButton/HomeButton';
 // props
+import Lever from '../Lever/Lever';
```

Comment:
Generally, I would prefer that you do not add to my code unless I say. I would rather the lever gets to final state and after that it may be incorporated into other sections of code.